src/alert/alert.scss

```diff
@@ -49,3 +50,48 @@
     width: 100vw;
   }
 }
+
+/// Modifier mixin to categorize the alert
+/// @arg $statusColor Can be one of: positive, negative, warning, primary
+/// @arg $rootSelector Selector of the root alert element. Defaults to .iui-alert
+@mixin iui-alert-category($statusColor, $rootSelector: '.iui-alert') {
```

Comment:
Do you use somewhere this with the differernt rootSelector? I dont think it is needed then here as an arg..

Reply:
I like it this way because it makes it easy to change it if needed, and does not closely tie the mixin to the class name.

Reply:
Could just had a variable with same outcome,  I guess.. but its minor. When somewhere previously you had mixin with 4-5 args for the same purpose, thats too many, IMO.